pkg/cloud/api/path.go

```diff
@@ -92,6 +147,20 @@ func (p Path) HasPrefix(prefix Path) bool {
 	return true
 }
 
+// isSliceIndex returns true if the element is a SliceIndex
+func isSliceIndex(element string) bool {
```

Comment:
Add unit tests to cover these new functions.

What is the coverage for the newly added code.